homeassistant/components/spotify/services.yaml

```diff
@@ -1,9 +0,0 @@
-play_playlist:
```

Comment:
Should we just keep a blank file here instead? I think if we remove the file entirely it triggers a warning on HA startup right?

Reply:
It didn't trigger a warning on my system, and I noticed there are many integrations that simply don't have this file (e.g., `buienradar` & `caldav`).

🤷‍♂ I'm happy to add an empty file though, but it seemed to me that was not needed.

Reply:
I could just be thinking of an old issue lol